index.html

```diff
@@ -504,7 +504,12 @@ <h2>
         at least its <a>origin</a>, including whether it is served over a
         secure connection. This UI SHOULD differ from any UI used when the
         [=Document/URL=] is <a>within scope</a>, in order to make it obvious
-        that the user is navigating off scope.
+        that the user is navigating off scope. In addition, the user agent
+        SHOULD reset the <a>application context</a>'s <a>default theme
```

Comment:
I don't like the term "reset" or "default theme color", since it implies making a permanent change to the default theme color itself. Rather, could it just say "SHOULD ignore the application's default theme color, instead using the theme color of the current document". (This could be more formal, but that's the idea.)